components/api/src/main/java/com/hotels/styx/api/extension/service/spi/AbstractStyxService.java

```diff
@@ -67,12 +72,17 @@ public StyxServiceStatus status() {
     @Override
     public CompletableFuture<Void> start() {
         boolean changed = status.compareAndSet(CREATED, STARTING);
+        LOGGER.info("Starting {}", name);
 
         if (changed) {
             return startService()
                     .exceptionally(failWithMessage("Service failed to start."))
-                    .thenAccept(na -> status.compareAndSet(STARTING, RUNNING));
+                    .thenAccept(na -> {
+                        status.compareAndSet(STARTING, RUNNING);
+                        LOGGER.info("Started {}", name);
+                    });
         } else {
+            LOGGER.info("Starting {} failed", name);
```

Comment:
do we need a log line next to an exception?

Reply:
Well not necessarily. But let me add the service name to the exception message.